java/Oleksii-Riabov/src/com/zoolatech/lecture1/tasks/_10/Task10.java

```diff
@@ -0,0 +1,64 @@
+package com.zoolatech.lecture1.tasks._10;
+
+import java.util.Random;
+import java.util.Scanner;
+
+/**
+ * Write a program that simulates the guess a number game. The program should generate a random
+ * number between 1 and N (N should be read from the input) and the user needs to guess it.
+ * If the user enters a number less than guessed, the program needs to output “Larger”. If the
+ * user enters a number greater than guessed - the program should output “Smaller”.
+ * If the user guesses the number correctly - the program should output “That’s it!” and finish.
+ */
+
+public class Task10 {
+
+    public static void main(String[] args) {
+
+        randomGenerator();
+    }
+
+    public static void randomGenerator() {
+        int randomNumber;
+
+        Scanner scanner = new Scanner(System.in);
+
+        Random randomGenerator = new Random();
+
+        if(scanner.hasNextInt()){
+            randomNumber = randomGenerator.nextInt(scanner.nextInt());
```

Comment:
`nextInt` generates random values from 0 to N-1, but that's not what you need

Reply:
Update task with nextInt to generate random values form 0 to N.